examples/java/spring-boot/src/test/java/uk/gov/api/springboot/infrastructure/repositories/EmptyApiStorageTest.java

```diff
@@ -7,17 +7,15 @@
 
 class EmptyApiStorageTest {
 
+  private final EmptyApiStorage storage = new EmptyApiStorage();
```

Comment:
I'd say this should be a separate commit - we could also see if there are other chances to do this elsewhere?